helm-charts/README.md

```diff
@@ -63,6 +63,77 @@ sh utils/k8s-dashboard/uninstall.sh
 
 ### Ordino
 
+## Chart Requirements
```

Comment:
@blazhovsky How did you generate these tables? Is there a command for it?

Reply:
Answer: https://github.com/norwoodj/helm-docs

Reply:
I used [helm-docs](https://github.com/norwoodj/helm-docs) for this.